src/components/Header/PopupModal/NotificationModal/NotificationModal.jsx

```diff
@@ -80,7 +80,8 @@ const NotificationModal = ({ isVisible, left, right, bottom, top }) => {
             height="100%"
             bgColor="lighterGray"
             borderRadius="0 0 1rem 1rem"
-            color="normalBlack">
+            color="normalBlack"
+            type="button">
```

Comment:
```suggestion
            color="normalBlack">
```

NotificationModal PR에서 Button 컴포넌트가 아닌 새로운 컴포넌트를 만들어서 이 변경사항은 빼주셔야 할 것 같습니다

Reply:
일전에 의견나눈대로 Text의 경우 컴포넌트 분리가 무의미하므로 내부 구현하는 방향으로 수정하겠습니다.